From: Jeremy Bicha <jbicha@ubuntu.com>
Date: Thu, 13 Dec 2018 06:54:51 -0500
Subject: Revert "Update some a11y test results"

This reverts commit b26cdd6d27a8ea7bd808a8861182797de232d174.

This change made gtk+3.0 3.24.2-1 fail its tests

Reported: https://gitlab.gnome.org/GNOME/gtk/issues/1516
---
 testsuite/a11y/colorchooser.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/testsuite/a11y/colorchooser.txt b/testsuite/a11y/colorchooser.txt
index aab761c..e74a52e 100644
--- a/testsuite/a11y/colorchooser.txt
+++ b/testsuite/a11y/colorchooser.txt
@@ -911,7 +911,7 @@ window1
                 "push button"
                 parent: grid
                 index: 5
-                state: enabled focusable sensitive visible
+                state: enabled focusable sensitive
                 toolkit: gtk
                 <AtkComponent>
                 layer: widget
